utilities/console.py

```diff
@@ -56,16 +57,25 @@ def connect(self):
     def _connect(self):
         self.child.send("\n\n")
         if self.username:
-            self.child.expect(self.login_prompt, timeout=TIMEOUT_5MIN)
-            LOGGER.info(f"{self.vm.name}: Using username {self.username}")
-            self.child.sendline(self.username)
-            if self.password:
-                self.child.expect("Password:")
-                LOGGER.info(f"{self.vm.name}: Using password {self.password}")
-                self.child.sendline(self.password)
-
-        self.child.expect(self.prompt, timeout=150)
-        LOGGER.info(f"{self.vm.name}: Got prompt {self.prompt}")
+            patterns = ["login:", "Password:", self.prompt]
```

Comment:
there's no need to use the timeoutsampler here; see https://pexpect.readthedocs.io/en/stable/overview.html#special-eof-and-timeout-patterns 
make the flow easy to follow